chart/values.yaml

```diff
@@ -757,6 +757,80 @@ scheduler:
 
   env: []
 
+# Pools that will be added to Airflow (Keys are pools names and values are pools settings)
```

Comment:
We need to be _very_ clear here that this doesn't actually manage the pools, just does a one off, at time of deploy, import.